### ISSUE TITLE
Error in firefox when selecting the whole paragraph block

### ISSUE THREAD

**Issue by carakas** [codex-team/editor.js#926]
Hello,

I'm currently looking at integrating this in Fork CMS

When I'm selecting all text in a paragraph in firefox I get the following error
```
TypeError: this.items[n] is undefined
```

**Comment by carakas** [comment]
Just tested this. I only have this when I use the out of the box editor without any extra tools registered

**Comment by pdfrod** [comment]
I'm having the same issue on Google Chrome. Here's a minimum reproducible example:

```
<html>
  <head>
    <title>Hello</title>
  </head>
  <body>
    <div id="editorjs"></div>
    <script src="https://cdn.jsdelivr.net/npm/@editorjs/editorjs@latest"></script>
    <script>
      var editor = new EditorJS();
    </script>
  </body>
</html>
```
1. write some text (e.g., "Hi")
2. double-click on the text to select it -> error happens

**Comment by khaydarov** [comment]
@pdfrod @carakas thanks for the report. WIll be fixed in 2.16

**Comment by carakas** [comment]
awesome :D

**Comment by neSpecc** [comment]
Resolved by #966
